### ISSUE TITLE
new cmd: delete part word left/right

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#1169]
### Short Summary

There is already a cmd_moveWordPartLeft/Right.  Refactor that code to extract a cmd_deleteWordPartLeft/Right

**Comment by Naatan** [comment]
What do either of these commands do exactly? Genuinely curious.

**Comment by cgchoffman** [comment]
Yep sorry, that's not totally clear.  I had to try it to see what it does.  It looks for obvious divisions in a string and will move/delete to that location. I'll update the summary so someone doesn't have to scroll through this ticket to see what it's supposed to do.

**Comment by Naatan** [comment]
Ahh I see. Would "Delete Word Left/Right" basically be what you want? (ctrl+delete and ctrl+backspace).

**Comment by mitchell-as** [comment]
No, he wants to delete "parts" of words where parts are delimited by underscores or changes in word case.

**Comment by Naatan** [comment]
ahhh gotcha, thanks :)
